plugins/boltz/db.go

```diff
@@ -21,11 +23,17 @@ type BoltzDB struct {
 }
 
 func (b *BoltzDB) Connect(dbPath string) error {
+	if _, err := os.Stat(dbPath); errors.Is(err, os.ErrNotExist) {
```

Comment:
why is this needed. bolthold creates database if it does not exist.

Reply:
You're right, changed that to create directory (which does not necessary exist)